Makefile

```diff
@@ -57,6 +57,9 @@ test-gpu:
 test-intel-fpga:
 	$(ACTIVATE) $(PYTEST) $(PYTEST_ARGS) tests/pytorch/fpga/
 
+test-xilinx:
+	$(ACTIVATE) $(PYTEST) $(PYTEST_ARGS) tests/pytorch/fpga/xilinx/
```

Comment:
Could you please use pytest markers instead? They are defined into `pytest.ini`

Reply:
Should be addressed now :)